engine/src/main/java/org/terasology/engine/logic/characters/CharacterMoveInputEvent.java

```diff
@@ -80,8 +80,8 @@ public boolean isCrouching() {
         return crouching;
     }
 
-    public boolean isJumpRequested() {
-        return jumpRequested;
+    public boolean isJumping() {
+        return jumping;
```

Comment:
This changes public API and might break modules, for instance. Did you check the Omega workspace that this still works?

Reply:
I checked whether any code I changed affects any modules in and it didn't.